### ISSUE TITLE
PHP-Worker container uses PHP-CLI 7.0.9 despite workspace using 7.1.4

### ISSUE THREAD

**Issue by SchwarzOliver** [laradock/laradock#927]
### Info:
- Docker version (`$ docker --version`):  Docker version 17.05.0-ce, build 89658be
- Laradock commit (`$ git rev-parse HEAD`): ac895209f0cf9f4591f0780a715ec4ada84ed2d7
- System info (Mac, PC, Linux): Linux
- System info disto/version: Ubuntu 16.04.2

##### What seems to be going wrong?

Hello,
I'm using Dockerfile-71 for the workspace container which is working fine. php-fpm is version 7.1 and so is php-cli.

When i build the php-worker container to use supervisor for the artisan queue:work command, this container uses php-cli 7.0.9. This is a problem, since we write code which makes usage of the new 7.1 features. So the php-worker container should also use 7.1.

Please update the Dockerfile for php-worker accordingly.

Thanks.

**Comment by mikeerickson** [comment]
@winfried-van-loon Can you confirm/deny this one seeing as you have done a lot of work around this stuff (much appreciated).

**Comment by winfried-van-loon** [comment]
You got it! Will look into it when I can make the time.

**Comment by mikeerickson** [comment]
@winfried-van-loon Thanks 👍

**Comment by winfried-van-loon** [comment]
Well, I've confirmed the bug and there's not much I can do, sadly. It was added in #673, it works, however it does only use a predefined PHP version, it seems like.

```
docker-compose exec php-worker bash
```

Php worker's base image can be found here: 
https://hub.docker.com/r/nielsvdoorn/laravel-supervisor/

I'm not sure if there are any image ARG's we can set for this image to change the PHP version - I'd assume so, but I can't find docs for it anywhere.
